test/sagemaker_tests/huggingface/inference/integration/sagemaker/test_hub_model.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+import sagemaker
+from sagemaker.model import Model
+from sagemaker.predictor import Predictor
+from sagemaker.serializers import JSONSerializer
+from sagemaker.deserializers import JSONDeserializer
+
+from ...integration import model_dir
+from ...integration.sagemaker.timeout import timeout_and_delete_endpoint
+
+# TODO: remove
+os.environ["AWS_PROFILE"] = "hf-sm"  # setting aws profile for our boto3 client
+os.environ["AWS_DEFAULT_REGION"] = "us-east-1"  # current DLCs are only in us-east-1 available
```

Comment:
Why is the configuration of the `AWS_DEFAULT_REGION` environment variable necessary? The `sagemaker_session` object being used on the tests should already be regionalized to the region provided by the `pytest` command in the `--region` arg.

Reply:
In my local setup, I don't have a default "AWS" configuration and it failed without setting this, saying there is no image, even If I have set `--region`. Not sure why this is needed. Maybe you need something to override it in `Session()`